README.md

```diff
@@ -86,7 +85,10 @@ See the [LICENSE](LICENSE)
 For more information
 --------------------
 
-* http://genomicsandhealth.org/
-* [LICENSE](LICENSE)
-* [Google Groups - old](https://groups.google.com/forum/#!forum/ga4gh-dwg-containers-workflows)
-* [Google Groups - new](https://groups.google.com/a/genomicsandhealth.org/forum/#!forum/ga4gh-dwg-containers-workflows)
+* [GA4GH Cloud Work Stream](https://github.com/ga4gh/wiki/wiki) - the wiki and meeting notes for the workstream
+* APIs that we co-ordinate/meet with
+  * [DRS](https://github.com/ga4gh/wiki/wiki/Data-Repository-Service)
+  * [TES](https://github.com/ga4gh/wiki/wiki/Task-Execution-Service)
+  * [WES](https://github.com/ga4gh/wiki/wiki/Workflow-Execution-Service)
+* [Global Alliance for Genomics and Health](https://www.ga4gh.org/) - GA4GH's main page
+* [GA4GH Technical Alignment Sub Committee (TASC)](https://github.com/ga4gh/TASC) - we try to co-ordinate GA4GH API decisions here
```

Comment:
Perhaps we could add the link to the Slack board here: https://ga4gh.slack.com/

However, the way things are currently set up, I think we can't just add an open invitation (I think they will expire after 3 days), so we would need to mention a contact where people can request invite links from. Not sure what to put in that case. Perhaps we should rather raise this with TASC

Reply:
I've created an issue for that here: https://github.com/ga4gh/TASC/issues/44